packages/odata-service-writer/src/index.ts

```diff
@@ -108,7 +108,7 @@ async function generate(basePath: string, service: OdataService, fs?: Editor): P
     if (service.metadata) {
         // copy existing `ui5.yaml` as starting point for ui5-mock.yaml
         if (paths.ui5Yaml && ui5Config) {
-            const webappPath = await getWebappPath(basePath);
+            const webappPath = await getWebappPath(basePath, fs);
```

Comment:
I'm wondering if its worth adding a spy to an existing test to ensure the fs parameter is passed to `getWebappPath` i.e.
```
import * as projectAccess from '@sap-ux/project-access'
const getWebappPathMock = jest.spyOn(projectAccess, 'getWebappPathMock');
expect(getWebappPathMock).toHaveBeenCalledWith(expect.anything(), fs);
```
Or even if you can count the number of parameters passed is two.